RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py

```diff
@@ -1,12 +1,62 @@
 import FWCore.ParameterSet.Config as cms
 
+###########################################################
+## Working points for the 102X training
+###########################################################
+full_102x_chs_wp  = cms.PSet(
+    #4 Eta Categories  0-2.5 2.5-2.75 2.75-3.0 3.0-5.0
+
+    #Tight Id
+    Pt010_Tight    = cms.vdouble( 0.69, -0.35, -0.26, -0.21),
+    Pt1020_Tight   = cms.vdouble( 0.69, -0.35, -0.26, -0.21),
+    Pt2030_Tight   = cms.vdouble( 0.69, -0.35, -0.26, -0.21),
+    Pt3050_Tight   = cms.vdouble( 0.86, -0.10, -0.05, -0.01),
+
+    #Medium Id
+    Pt010_Medium   = cms.vdouble( 0.18, -0.55, -0.42, -0.36),
+    Pt1020_Medium  = cms.vdouble( 0.18, -0.55, -0.42, -0.36),
+    Pt2030_Medium  = cms.vdouble( 0.18, -0.55, -0.42, -0.36),
+    Pt3050_Medium  = cms.vdouble( 0.61, -0.35, -0.23, -0.17),
+
+    #Loose Id
+    Pt010_Loose    = cms.vdouble(-0.97, -0.68, -0.53, -0.47),
+    Pt1020_Loose   = cms.vdouble(-0.97, -0.68, -0.53, -0.47),
+    Pt2030_Loose   = cms.vdouble(-0.97, -0.68, -0.53, -0.47),
+    Pt3050_Loose   = cms.vdouble(-0.89, -0.52, -0.38, -0.30)
+)
+
+###########################################################
+## Working points for the 94X training
+###########################################################
+full_94x_chs_wp  = cms.PSet(
+    #4 Eta Categories  0-2.5 2.5-2.75 2.75-3.0 3.0-5.0
+
+    #Tight Id
+    Pt010_Tight    = cms.vdouble( 0.69, -0.35, -0.26, -0.21),
```

Comment:
the values appear to be the same.
Do we really need to copy-paste this fairly large number of values?
Simply ```full_94x_chs_wp = full_102x_chs_wp.clone()``` would work.
Same comment applies to the other points.

Reply:
These numbers are preliminary the same, but it will potentially change in the near future. (and could *not* be the same for all the years like now) Perhaps is better to include the structure now?

Reply:
Looking back, indeed the values of the working points are rather different for different training cases.
What are the targets in training?

If the development strategy is to derive the working points from some fixed numerical target (like x% efficiency or rejection), then there would be very likely no accidental agreement. This, however doesn't seem to be the case this time.

If the WPs are weakly defined [this time?] with a replacement based on judgement calls,
it would be OK to copy-paste if the older values can update independently.

However it sounds like the development strategy this time is to first introduce placeholders to be replaced after some later processing. 
[I'm judging from the history of the past updates and guessing from the symptoms of potentially broken WPs visible in the validation results].
If you feel like a shorter ```new_wp = old_wp.clone() #TEMPORARY ``` is less acceptable,
please at least add an explicit comment inline that these are to be replaced soon.